### DISCUSSION TITLE
Routing with ZMQ

### DISCUSSION THREAD

**Discussion by cueltschey** [srsran/srsRAN_Project#551] [General Help]
When using iperf3 to simulate traffic in a zmq setup, the traffic is sent to the eth0 instead of ogstun. How would I route it differently?

Here are my commands:
sudo ip ro add 10.45.0.1/16 via 10.53.1.2
sudo ip netns exec ue1 ip route add default via 10.45.0.1 dev tun_srsue
iperf3 -s -i 1
iperf3 -c 10.45.0.1 -i 1 -t 60

**Comment by pgawlowicz** [comment]
do you run a dockerized open5gs version?

**Comment by improperaffo** [comment]
Try running the client side this way `ip netns exec ue1 iperf3 -c 10.45.0.1 -i 1 -t 60`

**Comment by pgawlowicz** [comment]
@cueltschey Please check your routing table, maybe there is a more specific route (e.g., 10.45.0.1/24) that goes over eth0. In such case, this one will be used.
